src/starknet-wrappers.ts

```diff
@@ -212,6 +219,26 @@ export abstract class StarknetWrapper {
     ): Promise<ProcessResult>;
 
     public abstract getTransaction(options: TxHashQueryWrapperOptions): Promise<ProcessResult>;
+
+    protected prepareBlockQueryOptions(command: string, options: BlockQueryWrapperOptions): string[] {
+
+        const commandArr = [
+            command,
+            options?.hash ? "--hash" : "",
```

Comment:
It would be better clearer to have `if` statements which check if there is `hash` or `number` supplied and which push to the parameter array accordingly. Then there would also be no need for filtering empty things.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/97/commits/798702d114086f92961a1ebb3beed7daccafc97c